Create 2020-07-03-candidate-level-english.md (#176)
DFE-Digital/bat-design-history: app/posts/apply-for-teacher-training/2020-07-03-candidate-level-english.md

But what about international candidates, who are less likely to have one?

[We’ve tried one design to help providers assess international candidates’ level of English, but research suggested that it was not quite right](/apply-for-teacher-training/international-candidates/#english-language-proficiency).

[Our new design](/apply-for-teacher-training/candidate-level-english/#our-new-design) has a different focus: asking international candidates to give details about their English language qualifications.

Why? To give providers evidence-based information about candidates, no matter where they’re from.

The challenge: designing something that works for everyone.

## Our new design
[We’re asking anyone who is not British or Irish if they have any ‘English as a foreign language’ qualifications](/apply-for-teacher-training/international-candidates/#english-as-a-foreign-language).

### User needs

{% from "user-needs/macro.njk" import appUserNeeds %}
{{ appUserNeeds({ items: collections['user-need'] | slugs(tags)}) }}

> ### Problems with this design
> Showing [the 'English as a foreign language' question](/apply-for-teacher-training/international-candidates/#english-as-a-foreign-language) only to candidates who are not British or Irish means we’re treating candidates differently depending on where they’re from.

Reviewer: ```suggestion
### Problems with this design

Showing [the 'English as a foreign language' question](/apply-for-teacher-training/international-candidates/#english-as-a-foreign-language) only to candidates who are not British or Irish means we’re treating candidates differently depending on where they’re from.
```

Finally, the question has the potential to deter candidates whose first language is not English from applying.

## Other ways to address this design challenge

### Bringing all English qualifications into one section

We could bring the ‘English GCSE or equivalent’ and ‘English as a foreign language’ questions into one question - which every candidate must answer.

This would not disrupt our existing designs very much.

For example:

{% from "email/macro.njk" import appEmail %}
{{ appEmail({
  content: "

> ### English GCSE or equivalent, or other evidence of ability in English

> What type of English qualification do you have?

> Select all that apply.

> [  ] GCSE

> [  ] O level

> [  ] Scottish National 5

> [  ] English Language Test
>For example, IELTS, LanguageCert International ESOL,TOEFL, Duolingo English Test, CPE,  CAE

> [  ] Other

> [  ] I do not have an English qualification
> Talk to your provider about showing your level of English.
"
}) }}

> This approach does not make assumptions about a candidate's ability based on their background.

Reviewer: ```suggestion
This approach does not make assumptions about a candidate’s ability based on their background.
```

> It also does not imply that one qualification is more valuable than another. (International candidates may not even realise that their language qualification is equivalent to an English GCSE.)

Reviewer: ```suggestion
It also does not imply that one qualification is more valuable than another. International candidates may not even realise that their language qualification is equivalent to an English GCSE.
```

### Asking candidates to indicate their own ability

> We also considered asking candidates to indicate their own ability on a proficiency scale, but quickly dismissed this idea.

Reviewer: ```suggestion
We also considered asking candidates to indicate their own ability on a proficiency scale, but dismissed this idea.
```
